Source/AtomicEditor/Editors/SceneEditor3D/SceneEditor3D.h

```diff
@@ -117,6 +120,8 @@ class SceneEditor3D: public ResourceEditor
 
     void UpdateGizmoSnapSettings();
 
+    SharedPtr<SystemUI::DebugHud> debugHud_;
+
```

Comment:
Hm, the SceneEditor holding a strong ref to the debugHud, let's make this a WeakRef, and would rather not require the member at all